src/permissionGroups/views/PermissionGroupDetails/PermissionGroupDetails.tsx

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import useNavigator from "@saleor/hooks/useNavigator";
+import useShop from "@saleor/hooks/useShop";
+import useBulkActions from "@saleor/hooks/useBulkActions";
+import createDialogActionHandlers from "@saleor/utils/handlers/dialogActionHandlers";
+import { maybe } from "@saleor/misc";
+import { useIntl } from "react-intl";
+import useNotifier from "@saleor/hooks/useNotifier";
+import { DEFAULT_INITIAL_SEARCH_DATA } from "@saleor/config";
+import { Button } from "@material-ui/core";
+import PermissionGroupDetailsPage from "../../components/PermissionGroupDetailsPage";
+import AssignMembersDialog from "../../components/AssignMembersDialog";
+import UnassignMembersDialog from "../../components/UnassignMembersDialog";
+import { usePermissionGroupDetailsQuery } from "../../queries";
+import {
+  usePermissionGroupAssignUsers,
+  usePermissionGroupUpdate,
+  usePermissionGroupUnassignUsers
+} from "../../mutations";
+import useStaffMemberSearch from "../../../searches/useStaffMemberSearch";
+
+import {
+  permissionGroupDetailsUrl,
+  PermissionGroupDetailsUrlQueryParams,
+  PermissionGroupDetailsUrlDialog,
+  permissionGroupListUrl
+} from "../../urls";
+import { PermissionGroupAssignUsers } from "../../types/PermissionGroupAssignUsers";
+import { PermissionGroupUnassignUsers } from "../../types/PermissionGroupUnassignUsers";
+import { PermissionGroupUpdate } from "../../types/PermissionGroupUpdate";
+
+interface PermissionGroupDetailsProps {
+  id: string;
+  params: PermissionGroupDetailsUrlQueryParams;
+}
+
+export const PermissionGroupDetails: React.FC<PermissionGroupDetailsProps> = ({
+  id,
+  params
+}) => {
+  const navigate = useNavigator();
+  const shop = useShop();
+  const notify = useNotifier();
+  const intl = useIntl();
+
+  const { data, loading, refetch } = usePermissionGroupDetailsQuery({
+    displayLoader: true,
+    variables: { id }
+  });
+
+  const { search, result: searchResult, loadMore } = useStaffMemberSearch({
+    variables: DEFAULT_INITIAL_SEARCH_DATA
+  });
+
+  const handleAssignUsersSuccess = (data: PermissionGroupAssignUsers) => {
+    if (data.permissionGroupAssignUsers.errors.length === 0) {
+      notify({
+        text: intl.formatMessage({
+          defaultMessage: "Users assigned"
+        })
+      });
```

Comment:
I believe that we should use `commonMessages.savedChanges` message rather than custom one.

Reply:
@dominik-zeglen even if it breaks parity with design?

Reply:
Also - IMO its UX driven decision to keep buttons title same as performed action. (when removing items we ask for "remove" not "save changes")

Reply:
We're talking about notifications, not buttons